projects/route-analyzer/CHANGELOG.MD

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Fix route-analyzer to use absolute paths instead of basenames
```

Comment:
Please add the header  and include something that helps the user know what happened instead of a description of the code change.


```
### [Fixed]
Fixed bug that broke the route-analyzer when two routing modules had the same base name
```


`### Fixed`